bin/cle-test-wrapper

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+#"$GENOMECI_BASEDIR"/bin/model-process-common $1 cle
+/gscuser/aregier/git/genome/fix/jenkins/bin/model-process-common $1 cle
```

Comment:
This should probably switch to the `$GENOMECI_BASEDIR` version.  (Should it also borrow the `if test -z "$GENOMECI_BASEDIR"` block from `model-process-common`?)